packages/common/src/hooks/usePageTitle/usePageTitle.tsx

```diff
@@ -0,0 +1,15 @@
+import { useEffect, useState } from 'react';
+import { useTranslation } from '@common/intl';
+
+export const usePageTitle = (title: string) => {
+  const t = useTranslation('common');
+  const [pageTitle, setPageTitle] = useState(title);
+
+  useEffect(() => {
+    const newTitle = `${title} | ${t('app')}`;
```

Comment:
We happy with this title format?

Reply:
yup. looks very slick next to this site 😁 

![Screen Shot 2022-03-29 at 10 18 15 AM](https://user-images.githubusercontent.com/9192912/160488899-e377fe36-6759-4043-a9b9-69e3042e1347.png)